toolchain/sem_ir/formatter.cpp

```diff
@@ -496,6 +497,10 @@ class InstNamer {
                            ".decl");
           continue;
         }
+        case LazyImportRef::Kind: {
+          add_inst_name("lazy_import_ref");
```

Comment:
Please also update `FormatInstructionLHS` to print the name of the instruction.

Also, we now have three of these (classes, interfaces, and lazy imports), so I think it's time to extend the infrastructure to allow tagging instructions as "has a name but no type" rather than adding special-case code to the formatter each time, but that's probably best done in a separate PR.

Reply:
Done